src/passes/J2CLOpts.cpp

```diff
@@ -154,54 +188,97 @@ class ConstantHoister : public WalkerPass<PostWalker<ConstantHoister>> {
   }
 
   AssignmentCountMap& assignmentCounts;
+  TrivialFunctionMap& trivialFunctionMap;
 };
 
-// A visitor that marks trivial once functions for removal.
-// TODO: parallelize
-class EnableInline : public WalkerPass<PostWalker<EnableInline>> {
+class TrivialOnceFunctionCollector
```

Comment:
This is no only collecting any more.

Reply:
Could you suggest a better name? It is still the purpose to collect. Let me see if removing the cleanup now changes anything....

Reply:
I think we can go back to just collecting.  I does not seem to have any impact, I would imagine that running `-O3` before the pass sort of guarantees that the cleanup afterward is a nop.